src/SparkPayloadEthereum.sol

```diff
@@ -38,4 +71,23 @@ abstract contract SparkPayloadEthereum is
             withDelegatecalls
         ));
     }
+
+    function _onboardAaveToken(address token, uint256 depositMax, uint256 depositSlope) internal {
+        SparkLiquidityLayerHelpers.onboardAaveToken(
+            Ethereum.ALM_RATE_LIMITS,
+            token,
+            depositMax,
+            depositSlope
+        );
+    }
+
+    function _onboardERC4626Vault(address vault, uint256 depositMax, uint256 depositSlope) internal {
```

Comment:
are we going to support similar operations on Base/Gnosis? does it make sense to define an interface/abstract contract and have different implementations of the same function depending on the network, similar to how we previously did for asserting ALM rate limits in base/mainnet?

Reply:
Yes which is why this is a convenience function around a library. I added the same function to Base.